Frameworks/UIKit/UIWebView.mm

```diff
@@ -243,11 +243,31 @@ - (void)setDetectsPhoneNumbers:(BOOL)detect {
 }
 
 /**
- @Status Interoperable
+ @Status Gap
 */
 - (NSString*)stringByEvaluatingJavaScriptFromString:(NSString*)string {
-    NSString* ret = [_xamlWebControl invokeScript:@"eval" arguments:[NSArray arrayWithObject:string]];
-    return ret;
+    UNIMPLEMENTED_WITH_MSG(
+        "stringByEvaluatingJavaScriptFromString is not supported on our platform. Call evaluateJavaScript:completionHandler: instead.");
+    return StubReturn();
```

Comment:
Thanks, no problem.  I'm not sure why it didn't fail for you; it failed on both debug and release CI builds.

---
In reply to: [95416396](https://github.com/Microsoft/WinObjC/pull/1638#discussion_r95416396) [](ancestors = 95416396)